CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 1.7.0-beta.1
+
+## ✨ Features
+* Add option to use a feature property as ID for feature state ([#8987](https://github.com/mapbox/mapbox-gl-js/pull/8987))
+* Use `browser.devicePixelRatio` instead of directly calling `window.devicePixelRatio` to be able to override it ([#9063](https://github.com/mapbox/mapbox-gl-js/pull/9063))
```

Comment:
What is the developer facing change here? Is it that developers can hijack `window.devicePixelRatio` and GL-JS will correctly get that update? cc @kkaefer 

Also missing h/t to pakastin

Reply:
Fixed the h/t. I had misunderstood the user facing change here a bit, not exactly sure how to summarize. My best understanding is that putting this on `browser` means you can override it, since it's not a read-only property on the window. I think this allows you to hack around an independently scaled map element to fix other downstream issues, which we are also working on supporting as separate features and fixes. Any advice here?

Reply:
Ok, I gave this another try to say more about why someone would use this. Good for beta?

Reply:
Confused about how to word this and the intention of the patch, sorry. Going to remove this for now and we can add back in the final release if needed.